app/views/test_executions/results/_subnav.html.erb

```diff
@@ -0,0 +1,15 @@
+<div>
+  <div class = "subnav navbar">
+    <div class = "navbar-inner">
```

Comment:
Let's name this partial along with the classname with something more descriptive. XMLNav or something like that? And does it need the extra wrapper div?